using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -126,6 +118,75 @@ Set the Push Identifier request
+
+### Audience Manager Content Response
+
+The Identity extension listens for the Audience extension response content event for any user opt-out updates. If this response contains `optedouthitsent: false` an opt-out network call is sent to the Experience Cloud Identity Service.
+
+For more details see [Audience Manager Content Response Document](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-audience-manager/audience-manager-event-reference#audience-manager-content-response-1).
+
+### Configuration Response Content
+
+#### Event Description
+
+Whenever the configuration extension dispatches the Configuration Response Content event, the Identity extension fetches the URL of the Experience Cloud Identity Service,  Experience Cloud Organization Identifier, and the Privacy status settings.
+
+This event will be generated in the following scenarios:
+
+- Initial config requested by the customer.
+- Configuration modified (remote update or local developer action).
+- In response to a configuration request event with the `config.getData` data key.
+
+For more information about the data payload definition for this event, see the *Configuration Keys* topics in each extension section.
+
+#### Event Details
+
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+The data property in the Configuration Response Content event is used by each extension to modify its current settings. Each extension is responsible to read out the part of the data property for which it is concerned.
+
+#### Data Payload Definition
+
+The Identity extension will read the following keys from the configuration event:
+
+Here are the key-value pairs in this event:
+
+| **Key**                  | **Value Type** | **Optional** | **Description**                                              |
+| :----------------------- | :------------- | :----------- | :----------------------------------------------------------- |
+| `experienceCloud.org`    | String         | Yes          | Experience Cloud Org Identifier                              |
+| `experienceCloud.server` | Sting          | Yes          | Custom endpoint to be used for Visitor ID Service network requests |
+| `global.privacy`         | String         | Yes          | Contains the mobile privacy status settings.                 |
+
+
+### Event Hub Shared State
+
+#### Event Description
+
+The shared state event is dispatched by the event hub after a shared state is created or updated. The event data contains the event owner, the name of the extension to which the shared state belongs. Upon hearing this event, the relevant listener rergistered by the Identity extension triggers processing of Identity module's event queue.
```

Comment:
I would rephrase the last sentence to: `The Identity extension triggers processing the queued events, if any, when the Configuration's shared state is updated.`